devmaster/modules/devshop/devshop_projects/drush/deploy.devshop.provision.inc

```diff
@@ -27,41 +27,42 @@ function drush_devshop_provision_pre_provision_devshop_deploy($git_ref = ''){
 /**
  * Implements the provision-devshop-deploy command.
  */
-function drush_devshop_provision_provision_devshop_deploy($git_ref = '')
-{
+function drush_devshop_provision_provision_devshop_deploy($git_ref = '') {
```

Comment:
All of the provision-devshop-deploy code needs to obe removed. The latest 1.x branch does not use it anymore. See devshop_hosting_hosting_tasks(), there is no longer a devshop-deploy task.